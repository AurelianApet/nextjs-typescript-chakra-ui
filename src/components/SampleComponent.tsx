import { Box, Button } from '@chakra-ui/react'

const SampleComponent = () => {
  return (
    <Box>
      <Button
        type="button"
        onClick={() => {
          return console.log('hello')
        }}
      >
        Test Button
      </Button>
    </Box>
  )
}

export default SampleComponent
