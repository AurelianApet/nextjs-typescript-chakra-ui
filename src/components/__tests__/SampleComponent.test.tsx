import { render } from '@testing-library/react'

import SampleComponent from 'components/SampleComponent'

describe('SampleComponent.tsx component', () => {
  const { container } = render(<SampleComponent />)

  test('match to snapshot', () => {
    expect(container).toMatchSnapshot()
  })
})
