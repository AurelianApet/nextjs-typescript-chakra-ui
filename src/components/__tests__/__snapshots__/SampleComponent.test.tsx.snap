// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleComponent.tsx component match to snapshot 1`] = `
<div>
  <div
    class="css-0"
  >
    <button
      class="chakra-button css-0"
      type="button"
    >
      Test Button
    </button>
  </div>
</div>
`;
